src/main/java/unicash/model/budget/Interval.java

```diff
@@ -0,0 +1,59 @@
+package unicash.model.budget;
+
+import static java.util.Objects.requireNonNull;
+import static unicash.commons.util.AppUtil.checkArgument;
+
+import unicash.commons.enums.BudgetInterval;
+
+/**
+ * Represents a Transaction's interval.
```

Comment:
I'm not exactly sure what this means, does it mean like the Interval between two transactions? i.e. one day/week/month between each transaction?

Reply:
Actually a typo. Supposed to be Budget interval